deimos-ruby.gemspec

```diff
@@ -25,6 +25,7 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency('sigurd', '>= 0.1.0', '< 1.0')
 
   spec.add_development_dependency('activerecord-import')
+  spec.add_development_dependency('activerecord-trilogy-adapter', '> 0')
```

Comment:
question: How come `> 0`? I haven't seen this before.

Reply:
Probably don't need it, it's the same as the above line that just has no condition.